eZ/Publish/API/Repository/Tests/ContentServiceTest.php

```diff
@@ -6687,4 +6687,21 @@ public function testPublishVersionWithSelectedLanguages()
             $content->fields['name']
         );
     }
+
+
```

Comment:
CS: extra blank line. Please run cs-fixer (still working on restoring automated checks)

Reply:
cs-fixer edits the whole file and this is not what I want to achieve. Instead, I fixed it by simply removing the line.

Reply:
> cs-fixer edits the whole file and this is not what I want to achieve. Instead, I fixed it by simply removing the line.

That is the whole point of cs-fixer :D If there are more issues, you should fix them all. Of course only for files you've modified.